includes/rest-api/class-sensei-rest-api-course-progress-controller.php

```diff
@@ -74,6 +74,7 @@ public function batch_delete_items( WP_REST_Request $request ) {
 			$student               = new WP_User( $student_id );
 			$result[ $student_id ] = false;
 			if ( $student->exists() ) {
+				$result[ $student_id ] = [];
```

Comment:
```suggestion
				$result[ $student_id ] = array();
```
I think we should use the old-style syntax for arrays to follow WP code style.

Reply:
Thanks! It's fixed in https://github.com/Automattic/sensei/pull/7650/commits/6a2ac9bc3cb1b869b37513010acb993f4fda4ba4.